SLHCUpgradeSimulations/Configuration/python/combinedCustoms.py

```diff
@@ -13,13 +13,13 @@
 import SLHCUpgradeSimulations.Configuration.aging as aging
 
 def cust_2017(process):
-    process=customisePostLS1(process)
+    process=customisePostLS1(process,displayDeprecationWarning=False)
```

Comment:
just a note: this is not a real change.
github web got confused with the merges